src/components/common/Extensions/CalamityBanner/CalamityBanner.tsx

```diff
@@ -45,7 +45,7 @@ const CalamityBanner: FC<PropsWithChildren<CalamityBannerProps>> = ({
               <Button
                 type="button"
                 className="md:mr-[1.6875rem] ml-4"
-                mode="primaryOutline"
+                mode="primaryOutlineFulled"
```

Comment:
it doesn't sounds good, why this change?

Reply:
Primary solid:
![image](https://github.com/JoinColony/colonyCDapp/assets/119701775/b5f857b5-28de-4a14-930d-156cb1f570ec)

Primary outline:
![image](https://github.com/JoinColony/colonyCDapp/assets/119701775/a0bbf97f-2cbe-4be2-9d0f-abe97fe0a770)

Primary outline fulled (notice that hover state is fully bg blue):
![image](https://github.com/JoinColony/colonyCDapp/assets/119701775/63466f79-5fee-4f33-9159-91585fb438a2)

Each of these buttons can have an icon and size default or small.
Do you have any idea how we can name the current 'primaryOutlineFulled' button better?